assets/js/custom.js

```diff
@@ -466,11 +471,5 @@ jQuery(document).ready(function() {
             let event = new Event('change');
             element.dispatchEvent(event);
             $('#table-main').css('display', 'block');
-            $( 'a' ).tooltip({
```

Comment:
This was the problematic tooltip usage that was causing the navbar tabs not to open and close properly. 

I think what happened here was this was a non-standard way of activating the pretty bootstrap tooltips, so for now, it will use browser-based tool tips. Later we can try and get the bootstrap ones working again, but I found the documentation (https://getbootstrap.com/docs/5.2/components/tooltips/) a bit confusing

Reply:
Ok!